### PR TITLE
Export Svg components in the Svg/index.js file

### PR DESCRIPTION
**Pull request by saschazar21**
This PR exports all `Svg` components (`Circle`, `Rect`, etc...) from the `lib/components/Svg/index.js` file.

This not only supports tree-shaking, it also improves the compatibility towards the `react-native-svg` API (especially when using tools like [SVGR](https://github.com/smooth-code/svgr) in combination with TypeScript).

Fixes: https://github.com/airbnb/react-sketchapp/issues/420

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,6 +44,7 @@
     "@skpm/sketchapp-json-plugin": "^0.2.0",
     "airbnb-prop-types": "^2.10.0",
     "error-stack-parser": "^2.0.0",
+    "gitbook-plugin-github": "^2.0.0",
```
Comment (mathieudutour) [inline package.json]: That shouldn't be here

Comment (saschazar21) [reply package.json]: Interestingly this dependency snuck in when I was running `yarn docs:watch`. Will remove it, however I can't guarantee it's not coming back some time in the future at someone else's place.

I think it's caused by this line in the `book.json`: https://github.com/airbnb/react-sketchapp/blob/master/book.json#L8

Comment (saschazar21) [reply package.json]: fixed in https://github.com/airbnb/react-sketchapp/pull/422/commits/8758f3783130a9138b1e40f4bd8dbe0ef6fcd5a6

**Comment by mathieudutour** [comment]
Thanks!

**Pull request merged**